package.json

```diff
@@ -67,7 +67,7 @@
       "git add"
     ],
     "src/**/*.go": [
-      "golangci-lint run -c .golangci.yml --fix",
```

Comment:
According to lint-staged document, this would be called as `golangci-lint run -c .golangci.yml --fix src/app/backend/handler/apihandler.go src/app/backend/integration/metric/heapster/client_test.go`. But this failed like follow if multiple file paths was passed to golangci-lint:
`ERRO Running error: context loading failed: no go files to analyze `

If single file path was passed to golangci-lint, the check was success.

Reply:
I said that there is a bug there :) We have to remove it for now.

Reply:
`golangci-lint` with `--fix` was removed temprarily by #3676.
This issue is separated to #3656 and kept to investigate.